content/doc/book/installing/kubernetes.adoc

```diff
@@ -85,6 +85,9 @@ kube-public       Active   50m
 kube-system       Active   50m
 ----
 
+
+If an error occurs you can just refer to our community page: https://community.jenkins.io/t/running-minikube-cluster-as-jenkins-agent
+
```

Comment:
@halkeye I'm interested in your opinion on this one.  I don't think that the conversation in https://community.jenkins.io/t/running-minikube-cluster-as-jenkins-agent provides enough information to justify a link from the documentation to that page.  I'd prefer that we not include this reference.

What are your thoughts on the level of content we should have on https://community.jenkins.io/ before we reference it on www.jenkins.io ?
```suggestion
```